zerver/openapi/zulip.yaml

```diff
@@ -2838,6 +2841,506 @@ paths:
+                                      type: integer
+                                      description: |
+                                        The default [message retention policy](/help/message-retention-policy)
+                                        for this organization.  Pass `"forever"` to request that messages
+                                        by retained forever (the default).
+                                    realm_name:
+                                      type: string
+                                      description: |
+                                        The name of the organization, used in login pages etc.
+                                    name_changes_disabled:
+                                      type: boolean
+                                      description: |
+                                        Indicates whether users are
+                                        [allowed to change](/help/restrict-name-and-email-changes) their name
+                                        via the Zulip UI in this organization.  Typically disabled
+                                        in organizations syncing this this type of account information
+                                        an external user database like LDAP.
+                                    avatar_changes_disabled:
+                                      type: boolean
+                                      description: |
+                                        Indicates whether users are
+                                        [allowed to change](/help/restrict-name-and-email-changes) their avatar
+                                        via the Zulip UI in this organization.  Typically disabled
+                                        in organizations syncing this this type of account information
+                                        an external user database like LDAP.
+                                    emails_restricted_to_domains:
+                                      type: boolean
+                                      description: |
+                                        Whether [new users joining](/help/allow-anyone-to-join-without-an-invitation)
+                                        this organization are required to have an email
+                                        address in one of the `realm_domains` configured for the organization.
+                                    send_welcome_emails:
+                                      type: boolean
+                                      description: |
+                                        Whether or not this organization is configured to send the standard Zulip
+                                        [welcome emails](/help/disable-welcome-emails) to new users joining the organization.
+                                    ? realm_message_content_allowed_in_email_notifications
+                                    : type: boolean
```

Comment:
This couple lines seem to be buggy; I'm not sure how an OpenAPI parser reads this and thus whether we could reasonably expect our tooling to have caught this.

Reply:
It appears that `prettier` re-adds this bug if one fixes it manually, so there's something more complicated going on here.

@andersk FYI in case you have any ideas.

Reply:
@timabbott According to https://stackoverflow.com/questions/33987316/what-is-a-complex-mapping-key-in-yaml#:~:text=On%20the%20YAML%20specs%20there,%2C%20colon%2C%20or%20question%20mark. it is a valid yaml syntax. I think Prettier is changing the syntax because of the length of the key(it doesn't change the syntax if I reduce the length of the key), but I'm not sure why it hadn't happened for the same key earlier in documentation.
Also, this had to be changed to `message_content_allowed_in_email_notifications` from `realm_message_content_allowed_in_email_notifications` but it missed by eye because of the different syntax and resulting non-highlighting. Apologies!

Reply:
@timabbott This is indeed happening due to the long property name(along with the preceding tabs) as per https://github.com/prettier/prettier/issues/5599. A workaround could be modifying the Prettier config file to allow longer length?

Reply:
Hmm, if it works, I'm OK with it.  Perhaps the right actual fix might be to move some of this content to be in a component, so that it isn't so deeply intended.

Reply:
Though it is a bit puzzling that if we make this line wrong (with the `realm_` prefix), the tests don't fail.